src/commands/which.ts

```diff
@@ -12,6 +18,8 @@ export default class Which extends Command {
       // If this if statement is true then the command to find was passed in as a single string, e.g. `mycli which "my command"`
       // So we must use the topicSeparator to split it into an array
       command = argv[0].split(this.config.topicSeparator)
+    } else {
+      throw new Error('Missing 1 required arg')
```

Comment:
What would be a more helpful message for the user?  Something like:

```suggestion
      throw new Error('"which" expects a command name.  Try something like "which your:command:here" ')
```